gcs/CONFIGURATION.md

```diff
@@ -22,7 +22,7 @@
     Whether to perform copy operation using Rewrite requests which allows copy
     files between different locations and storage classes.
 
-*   `fs.gs.rewrite.max.bytes.per.call` (default: `536870912`)
+*   `fs.gs.rewrite.max.chunk.per.call` (default: `512m`)
 
     Maximum number of bytes rewritten in a single rewrite request when
```

Comment:
Do we have to change the description to match the same? i.e. in the description, we are mentioning bytes and the value can be in MB for e.g.

Reply:
Done